docs/guides/deprecating-content.rst

```diff
@@ -78,11 +78,10 @@ This won't hide results from that page, but it will give priority to results fro
 
 .. TODO: mention search.ignore when it's implemented.
 
-
 .. tip::
 
-   If you are using Sphinx with reStructuredText,
-   you can make use of some :doc:`directives <sphinx:usage/restructuredtext/directives>`
+   You can make use of Sphinx :doc:`directives <sphinx:usage/restructuredtext/directives>`
+   or MkDocs `admonitions <https://python-markdown.github.io/extensions/admonition/>`_
```

Comment:
think this should be bellow the list of sphinx directives (`deprecated`, `versionchanged`...)

Reply:
Fair enough, I was hesitating about it as well. Will change it and merge.